Snake/BusinessLayer/GameSnake.cs

```diff
@@ -38,12 +41,12 @@ public List<Point> createFirstSnake(int width, int height, int newLength)
 
 
         public List<Point> snakeMove(int direction, Boolean hasEaten)
-        {
-            //make new list equal to position to previous state of snake
-            List<Point> snakeMoveList = currentPosition;
-            Point snakeTurn = new Point(0, 0);
-            //.directionFacing = direction;
-
+        {
+            //make new list equal to position to previous state of snake

```

Comment:
this is better comment just needs a bit more of the why -> we make new list equal to position to previous state of snake because it will make snake appear on opposites side of the screen....